services/order.py

```diff
@@ -0,0 +1,32 @@
+from django.contrib.auth import get_user_model
+from django.db.models import QuerySet
+from db.models import Order, Ticket, MovieSession
+from django.db import transaction
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: int = None
+) -> None:
+    with transaction.atomic():
+        user = get_user_model().objects.get(username=username)
+        new_order = Order.objects.create(user=user)
+        if date:
+            new_order.created_at = date
+            new_order.save()
```

Comment:
So if you don't pass the date to the function, the order won't be saved or what? Move the save one level down, like this:
```suggestion
        if date:
            new_order.created_at = date
        new_order.save()
```